src/commons/crypto/signing/dispatch/krillsigner.rs

```diff
@@ -251,7 +251,7 @@ impl KrillSigner {
         let signed_and_encoded_csr = Csr::construct_rpki_ca(
             &self.router,
             key,
-            &base_repo.ca_repository(name_space).join(&[]).unwrap(), // force trailing slash
+            &base_repo.ca_repository(name_space),
```

Comment:
Note: the trailing slash is now enforced in both `ca_repository` and `Csr::construct_rpki_ca` in the rpki-rs 0.16.0.